README.md

```diff
@@ -1,6 +1,6 @@
-# Microsoft Azure CLI kubectl plugin
+# Microsoft AKS CLI kubectl plugin
```

Comment:
```suggestion
# Microsoft Azure Kubernetes Service (AKS) CLI kubectl plugin
```
WDYT?

Reply:
Agreed. Makes sense to use the complete name here.